### PR TITLE
Gemini result report

### PR DESCRIPTION
**Pull request by aleksbykov**
- adding results to elastic
- send email report with results from test

### CODE REVIEW

File: sdcm/sct_config.py [hunk 1]
```diff
@@ -291,10 +291,10 @@ class SCTConfiguration(dict):
                     be provided by the test suite infrastructure.
                     multiple commands can passed as a list"""),
 
-        dict(name="gemini_url", env="SCT_GEMINI_URL",  type=str,
-             help="""Url of download of the binaries of gemini tool"""),
+        dict(name="gemini_version", env="SCT_GEMINI_version",  type=str,
```
Comment (bentsi) [inline sdcm/sct_config.py]: Env variables should use uppper case

File: sdcm/sct_config.py [hunk 2]
```diff
@@ -291,10 +291,10 @@ class SCTConfiguration(dict):
...
+             help="""Version of download of the binaries of gemini tool"""),
 
-        dict(name="gemini_static_url", env="SCT_GEMINI_STATIC_URL",  type=str,
+        dict(name="gemini_schema_url", env="SCT_GEMINI_STATIC_URL",  type=str,
```
Comment (bentsi) [inline sdcm/sct_config.py]: `SCT_GEMINI_STATIC_URL` - > `SCT_GEMINI_SCHEMA_URL`

File: sdcm/tester.py [hunk 3]
```diff
@@ -225,7 +226,7 @@ def setUp(self):
             self.db_cluster.wait_for_init(node_list=self.db_cluster.nodes[:seeds_num])
             self.db_cluster.wait_for_init(node_list=self.db_cluster.nodes[seeds_num:])
         else:
-            self.db_cluster.wait_for_init()
+            self.db_cluster.wait_for_init(verbose=True)
```
Comment (bentsi) [inline sdcm/tester.py]: why did you change this?

File: sdcm/tester.py [hunk 4]
```diff
@@ -836,7 +843,27 @@ def get_stress_results_bench(self, queue):
...
     def get_gemini_results(self, queue):
         results = self.loaders.get_gemini_results(queue)
-        return results
+        result = self.verify_gemini_results(results)
+        return result
+
+    def verify_gemini_results(self, results):
+        stats = {'status': None, 'results': []}
+        if not results:
+            self.fail('Gemini results are not found.')
```
Comment (bentsi) [inline sdcm/tester.py]: AFAIK `self.fail` will throw an exception so all the code below will be unreachable.

File: tests/gemini_test.yaml [hunk 5]
```diff
@@ -13,9 +13,9 @@ store_results_in_elasticsearch: False
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
-gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
-gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+gemini_cmd: "gemini -d --duration 120s -c 20 -p 100 -m mixed -f"
```
Comment (bentsi) [inline tests/gemini_test.yaml]: why duration is different  from  `test-cases/gemini/gemini-basic-3h.yaml`?

File: sdcm/sct_config.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/sct_config.py]: Fixed

File: sdcm/sct_config.py (hunk 2 above)
Comment (aleksbykov) [reply sdcm/sct_config.py]: Fixed

File: sdcm/tester.py (hunk 3 above)
Comment (aleksbykov) [reply sdcm/tester.py]: returned back

File: sdcm/tester.py (hunk 4 above)
Comment (aleksbykov) [reply sdcm/tester.py]: replace with log.error

File: tests/gemini_test.yaml (hunk 5 above)
Comment (aleksbykov) [reply tests/gemini_test.yaml]: fixed

**Comment by bentsi** [comment]
Closing this, Alex opened updated PR #956
